### PR TITLE
Fix missingservice

### PR DESCRIPTION
**Pull request by ansmeer**
This PR fixes a major bug in the previous version that prevented the app from starting.

The missing bot service was not added directly due to a circular dependency. Instead, the relevant methods from bot service were refactored to the newly created embed service.

Superfluous module dependencies were removed and docs added to show an overview over all current modules and their dependencies.

### CODE REVIEW

File: docs/dev-guide/modules.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+CabinUtModule --> VisbookModule
+DiscordModule --> ConfigModule
+DiscordModule --> CabinUtModule
+DiscordModule --> SlashCommandModule
+EmbedModule
+SlashCommandModule --> ConfigModule
+SlashCommandModule --> CabinModule
+SlashCommandModule --> DiscordModule
+SlashCommandModule --> EmbedModule
+VisbookModule --> ConfigModule
```
Comment (gingermusketeer) [inline docs/dev-guide/modules.md]: Would it be worth dropping the config module as it just adds noise?

**Review by gingermusketeer — APPROVED** [review]
Thanks for the diagram. Looks like we could refactor things to avoid some of the circular references.

**Comment by ansmeer** [comment]
I believe there's only one circular dependency left since this fix gets rid of the recently introduced one. Will have a look at that next.

File: docs/dev-guide/modules.md (hunk 1 above)
Comment (ansmeer) [reply docs/dev-guide/modules.md]: I think that's a good idea, will remove it from the chart, it will make things easier to see.

**Pull request merged**